liblib/CMakeLists.txt

```diff
@@ -200,27 +232,27 @@ ENDIF(ENABLE_SHARED_LIBRARY_VERSION)
 # https://github.com/Unidata/netcdf-c/issues/135
 # for more information.
 ###
-IF(${CMAKE_VERSION} VERSION_LESS 2.8.12)
-  INSTALL(
+if(${CMAKE_VERSION} VERSION_LESS 2.8.12)
```

Comment:
Remove

Reply:
Reading the documentation on [`INCLUDES DESTIONATION`](https://cmake.org/cmake/help/latest/command/install.html#signatures), this one does not have the effect of `PUBLIC_HEADER`. The former signature is perfectly fine.